### DISCUSSION TITLE
OT: pdf2djvu

### DISCUSSION THREAD

**Discussion by jsbien** [FriedrichFroebel/didjvu#13] [General]
Would you be so kind to enable issues and/or discussions on your fork of pdf2djvu?

**Comment by FriedrichFroebel** [comment]
My current *pdf2djvu* fork rather is a mess at the moment, but as I do not see any problems with opening them, I just did.

**Reply by jsbien** [reply]
Thanks!

**Reply by FriedrichFroebel** [reply]
To add on this: I just re-created the fork from the latest upstream version. The ported test changes have to be re-applied (diff available) to perform some initial cleanup. I will see when I find the time to do so. Nevertheless, I do not have any direct plans to actually work on *pdf2djvu* like I do for *didjvu* and *ocrodjvu* (which I actually use), whereas I am not actively using *pdf2djvu*. That being said, the future of *pdf2djvu* is unclear to me, but I am up to support anyone wanting to further work on this.

**Reply by jsbien** [reply]
It's OK for me.
